src/balance.js

```diff
@@ -1,17 +1,33 @@
 const { BN } = require('./setup');
 
-async function balanceCurrent (account) {
-  return new BN(await web3.eth.getBalance(account));
+class Tracker {
+  constructor (acc) {
+    this.account = acc;
+    this.current = async () => new BN(await web3.eth.getBalance(this.account));
```

Comment:
Why not simply use `balanceCurrent`?

Reply:
I don't have a good reason, tbh. I think in my mental model `balance.current` and `balance.tracker` are two separate things and therefore it might make sense to make them independent. But I don't feel so strong about it though, so I'll just changing it!

Reply:
Done!